server/models/board.js

```diff
@@ -0,0 +1,55 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const BoardSchema = new Schema({
+  name: {
+    type: String,
+    trim: true,
+    required: true,
+  },
+  createdBy: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+    required: true,
+  },
+  createdAt: {
```

Comment:
use timestamp option as supported in moongoose.

Reply:
👍